legal/terms-of-service.md

```diff
@@ -48,39 +48,39 @@ as of 05/2018
 5. When using testing data the Customer will make sure that these data do not contain unanonymized personal data of real people.
 6. The Customer will indemnify and hold harmless Travis CI, its officers and directors, employees and agents from any and all third party claims, damages, costs and (including reasonable attorneys fees) arising out of the Customer’s use of the Travis CI Software in a manner not authorized by this Agreement, and/or applicable law, or the Customer’s or it’s employees’ or personnel’s negligence or willful misconduct.
 
-### Downtime and services suspensions
+### 5. Downtime and services suspensions
 
 1. Adjustments, changes and updates of the Travis CI Software that help to avoid or maintain dysfunctions of the Travis CI Software may lead to temporary service suspensions. Travis CI will try to limit downtime of the service or restrictions of accessibility to 10 hours a month. Travis CI will try to do regular maintenance works during the weekend or at times between 10 p.m. and 6 a.m. (CET). Outside of the said hours, Travis CI will inform the Customer about upcoming maintenance works.
 2. The Customer is aware that the service relies on a working internet infrastructure. Additional downtime of the service can occur if the website is not available and at any other time with restrictive access to the internet of the Customer.
 3. The Customer is aware that the Travis CI Software does not work if GitHub is not properly available (be it to Travis CI or the Customer).
 4. For the avoidance of doubt, in case of dysfunctions of the Travis CI Software caused by Travis CI through willful intent or gross negligence, Travis CI’s liability for such dysfunctions is not excluded by this section 5
 
-### Rights to use
+### 6. Rights to use
 
-1. TSubject to the full payment of due fees for the services under the Agreement, Customer is granted a limited, non-exclusive, non-transferable, non-sublicenseable right to use the Travis CI Software as software as a service via the internet according to these Terms and Conditions.
+1. Subject to the full payment of due fees for the services under the Agreement, Customer is granted a limited, non-exclusive, non-transferable, non-sublicenseable right to use the Travis CI Software as software as a service via the internet according to these Terms and Conditions.
```

Comment:
Please isolate this typo to a single PR, and remove the numbers. Thanks.

Reply:
Can do, but to be honest the main reason I opened this PR was because I found the ToS difficult to read. For example when I'm reading the "Scope of services" section, and it makes reference to "section 9", I had to scroll to the first heading, then count the headings until I got to 9. 

Also, the paragraphs within each section are numbered, so I figured it made sense to make this change so the reader can quickly find e.g. "section 5.1".

See also:
https://docs.travis-ci.com/legal/privacy-policy/#i-what-are-personal-data

Reply:
I understand the motivation. But hard coding the numbers makes it difficult to maintain. If we _do_ choose to number the sections in this way, the thing to do is to have an ordered list within an ordered list, instead of hard coding the numbers inside the section headers.

```markdown
1. ### Subject matter

    1. The subject matter of these terms and conditions (hereinafter **“Terms and Conditions”**) is the use of Travis CI Software. The Terms and Conditions regulate all relations between Travis CI and Travis CI’s customer (hereinafter **“Customer”**) regarding the use of Travis CI Software and are part of the Agreement as defined below. Standard business conditions and/or general terms and conditions of the Customer do not apply, regardless of whether or not Travis CI has expressly objected to them in a particular case.
    2. Travis CI communicates with the GitHub hosting service (hereinafter **“GitHub”**), which is offered by GitHub Inc. on the Customer’s behalf. Condition to the proper use of Travis CI Software is a valid contract with GitHub Inc. on the use of GitHub, which may lead to costs on the Customer’s sole responsibility. The Customer will provide Travis CI with his GitHub account information (hereinafter **“GitHub Sign-In”**) when signing in through GitHub via travis CI-ci.com (hereinafter the **“Website”**) automatically. He allows Travis CI to access the Customer’s GitHub account. Travis CI will directly communicate in the name of the Customer with GitHub and the Customer authorizes Travis CI to act on his behalf towards GitHub Inc. The Customer is solely liable for any costs or damages that GitHub Inc. associates with the GitHub Sign-In.

1. ### Scope of services

    1. Travis CI provides Travis CI Software as a service. This means, that Travis CI Software may be used via the internet only.
    2. The Travis CI Software is a hosted continuous integration service that provides the infrastructure for testing software projects subject to downtime and service suspensions as described below. The functional range of the service is dependent on the package the Customer chooses and will be displayed on the Website.
    3. Travis CI offers its services for (i) closed software projects, where the projects cannot be viewed publicly and (ii) open software projects, where the projects including their source codes can be viewed publicly.
    4. The Travis CI Software is integrated with GitHub and offers support for several programming languages. The entire list of supported programming languages is available at about.travis CI-ci.org.
    5. No consultancy, training, trouble shooting or support is within the scope of the services offered by Travis CI under this Agreement if and to the extent they are not part of the agreed warranty as set out in section 9 hereunder.
    6. Travis CI offers unpaid and paid services. If a service by Travis CI is offered free of charge, section 8 shall apply.
⋮
```

This would render like this:

![Screen Shot 2019-11-08 at 09 38 57](https://user-images.githubusercontent.com/25666/68484621-9f603680-020b-11ea-8e18-de2668e85874.png)

How do you feel about this, @Pezi777?

Reply:
Maybe, but I would say they are implicitly "hard coded" anyway, since the ToS text makes references to section numbers. So if you wanted to re-arrange sections, you would have to modify that text.

Reply:
These docs have plenty of hard coded numbering
```md
## Website

### II. How are my Data used when visiting the Website?

1. **Visiting the Website**

    We (or the webspace provider) collect data about each...
```
Just go with it, no point reference sections by numbers that don't exist